Branch-SDK-Automation-TestBed/src/main/java/io/branch/branchandroiddemo/BranchWrapper.java

```diff
@@ -250,4 +252,26 @@ public void onFailure(Exception e) {
             showLogWindow("Test Data : Null", true, ctx, Constants.CREATE_QR_CODE);
         }
     }
+
+    public void setAttributionLevel(Intent intent){
```

Comment:
We can shutter this whole app then, right?

Reply:
Yeah, we won't need this anymore

Reply:
Then I would say remove these updates from this pr.

Reply:
Got it, i'll just remove all the Appium testbed changes made